src/router.ts

```diff
@@ -632,6 +650,15 @@ const routeVoterToSlackChannel = async (
     // Remember the voter's thread in this channel.
     userInfo[response.data.channel] = response.data.ts;
 
+    // Create the thread
```

Comment:
can you add to the comment that this new thread will adopt the old thread's needAttention state?